test_scripts/Resumption/InteriorVehicleData/026_InteriorVD_resumption_double_subscription_with_moduleId_unexpected_disconnect.lua

```diff
@@ -0,0 +1,94 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: SDL sends GetInteriorVehicleData response with WARNINGS result code
+--  in case of double subscription with custom moduleId after successful resuming on transport disconnect
```

Comment:
@VjKlepikov I would propose to change description to be more accurate
```suggestion
--  in case of double subscription with specified moduleId after successful resuming on transport disconnect
```